packages/react-menu/src/stories/MenuMenuItemsWithIcons.stories.tsx

```diff
@@ -1,12 +1,7 @@
 import * as React from 'react';
-// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
+
 import { Menu, MenuTrigger, MenuList, MenuItem, MenuPopover } from '@fluentui/react-menu';
```

Comment:
```suggestion
import { Menu, MenuTrigger, MenuList, MenuItem, MenuPopover } from '../../index';
```

Should not be this something like this and use local exports?

Reply:
yeah, menu was one of the first packages to introduce these kinds of stories and lagged behind practices... will update